CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for the Mapbox Gestures for Android
 
+## 0.8.0 - 
```

Comment:
Missing a date.

Reply:
Yeah, haven't added it since wasn't sure I'll cut a release today ;)

Reply:
Scheduled for tomorrow